### PR TITLE
Run calculation results for all test cases and disply the results

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-3895](https://jira.cms.gov/browse/MAT-3895)
(Optional) Related Tickets:

### Summary

- Calculation results are compared with the expected values, and the status is displayed on the testCases list page.

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/routes/TestCaseRoutes.test.tsx [hunk 1]
```diff
@@ -135,12 +137,16 @@ describe("TestCaseRoutes", () => {
     const testCaseTitle = await screen.findByText("TC1");
     expect(testCaseTitle).toBeInTheDocument();
     const newBtn = screen.getByRole("button", { name: "New Test Case" });
-    userEvent.click(newBtn);
+    await waitFor(() => {
+      userEvent.click(newBtn);
+    });
```
Comment (jkotanchik-SB) [inline src/components/routes/TestCaseRoutes.test.tsx:142]: Turns out this is an anti-pattern. I'm working on adjusting it now.

File: src/components/routes/TestCaseRoutes.test.tsx [hunk 2]
```diff
@@ -86,8 +86,10 @@ describe("TestCaseRoutes", () => {
 
     const testCaseTitle = await screen.findByText("TC1");
     expect(testCaseTitle).toBeInTheDocument();
-    const newBtn = screen.getByRole("button", { name: "New Test Case" });
-    userEvent.click(newBtn);
+    await waitFor(() => {
+      const newBtn = screen.getByRole("button", { name: "New Test Case" });
+      userEvent.click(newBtn);
+    });
     const testCaseForm = screen.getByTestId("create-test-case-form");
```
[the remark this answers is not in the document]
Comment (jkotanchik-SB) [reply src/components/routes/TestCaseRoutes.test.tsx]: Fixed in my branch.

File: src/components/routes/TestCaseRoutes.test.tsx (hunk 1 above)
Comment (jkotanchik-SB) [reply src/components/routes/TestCaseRoutes.test.tsx]: Also fixed.

**Review by jkotanchik-SB — APPROVED** [review]
